pysrc/models/column_statistics.py

```diff
@@ -0,0 +1,13 @@
+from dataclasses import dataclass
+from dataclasses_json import dataclass_json
+
+@dataclass_json
+@dataclass(frozen=True)
+class ColumnStatistic:
+    count: int
+    unique_count: int
+    unique_ratio: float
+    min: str
+    max: str
+    shortest: str
+    longest: str
```

Comment:
These properties store actual values from the source. Just to make sure: This is the intended behavior?

Reply:
Yes, not sure if it will be useful, but there is no harm in saving them.